modules/back-end/src/Infrastructure/Experiments/ExperimentService.cs

```diff
@@ -49,8 +49,8 @@ public async Task ArchiveExperiment(Guid envId, Guid experimentId)
                 CustomEventTrackOption = (int)iteration.CustomEventTrackOption,
                 CustomEventSuccessCriteria = (int)iteration.CustomEventSuccessCriteria,
                 CustomEventUnit = iteration.CustomEventUnit,
-                StartExptTime = iteration.StartTime.ToString("yyyy-MM-ddTHH:mm:ss.ffffff"),
-                EndExptTime = iteration.EndTime?.ToString("yyyy-MM-ddTHH:mm:ss.ffffff")
+                StartExptTime = iteration.StartTime.Millisecond,
+                EndExptTime = iteration.EndTime.HasValue ? iteration.EndTime.Value.Millisecond : null
```

Comment:
https://learn.microsoft.com/en-us/dotnet/api/system.datetime.millisecond?view=net-6.0

In this file, there are many places u call `DateTime.Millisecond`, but its result is not what u expected. This should cause a BUG.

Reply:
done